lib/cli/src/Cardano/CLI.hs

```diff
@@ -1303,19 +1308,17 @@ tlsOption = TlsConfiguration
         <> help "The RSA Server key which signed the x.509 server certificate."
 
 smashURLOption
-    :: Parser URI
+    :: Parser PoolMetadataSource
 smashURLOption = option (eitherReader reader) $ mempty
-    <> long "smash-url"
-    <> metavar "URL"
-    <> help "Optional HTTP(S) address of a \
-            \Stakepool Metadata Aggregation Server."
+    <> long "pool-metadata-fetching"
+    <> metavar "STRATEGY"
+    <> help "Pool Metadata fetching strategy. This setting will persist across restarts. Possible values: \
+            \ none (default), \
```

Comment:
NOTE: there's a `default` combinator for optparse-applicative and also an option to show defaults in the help text auutomatically based on that value.

Reply:
It's called `value` and `showDefault`.